content/guides/models/app/features/panels/line-plot/smoothing.md

```diff
@@ -8,22 +8,30 @@ title: Smooth line plots
 weight: 30
 ---
 
-W&B supports three types of smoothing:
+W&B supports three types of back-end smoothing applied by W&B Server:
```

Comment:
```suggestion
W&B supports three types of back-end smoothing applied by W&B Server:
```

Only the exponential moving average smoothing algorithm (marked tensorboard below) has backend support as of my change. The other three types all are done exclusively on the frontend. The new feature I added is the backend support for EMA

Reply:
Done, PTAL